api/equipment/schema.py

```diff
@@ -3,12 +3,42 @@
 from graphene import relay, Schema
 from graphene_sqlalchemy import (SQLAlchemyObjectType, 
                                  SQLAlchemyConnectionField)
-
 from api.equipment.models import Equipment as EquipmentModel
+from utilities.utility import validate_empty_fields
 
 
 class Equipment(SQLAlchemyObjectType):
     
     class Meta:
         model = EquipmentModel
-        interfaces = (relay.Node, )
\ No newline at end of file
+        interfaces = (relay.Node, )
+
+class UpdateRoomResource(graphene.Mutation):
+    class Arguments:
+        name = graphene.String()
+        room_id = graphene.Int()
+        resource_id = graphene.Int()
+
+    equipment = graphene.Field(Equipment)
+    def mutate(self,info,room_id,resource_id,**kwargs):
```

Comment:
Would it be ideal to just identify a resource using the resource id and not a combination of resource ID and room ID, remember in updating room resource, we might do things like transfer a resource from one room to another